### PR TITLE
refactor: modernize package

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        update-types:
+          - patch
+  # dev-deps
+  - package-ecosystem: npm
+    schedule:
+      interval: weekly
+    allow:
+      - dependency-type: development
+    ignore:
+      - dependency-name: '@map-colonies/schemas'
```
Comment (netanelC) [inline .github/dependabot.yml:30]: Remove it

File: .github/workflows/release-pleate.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+on:
```
Comment (netanelC) [inline .github/workflows/release-pleate.yaml:1]: Change name of file

**Review by netanelC — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (CptSchnitz) [reply .github/dependabot.yml]: fixed

File: .github/workflows/release-pleate.yaml (hunk 2 above)
Comment (CptSchnitz) [reply .github/workflows/release-pleate.yaml]: fixed

**Review by netanelC — APPROVED** [review]

**Pull request merged**